bin/batch.sh

```diff
@@ -3,4 +3,4 @@ conf_file=${1:-"conf/batch.yaml"}
 
 set -x
 
-java ${JAVA_OPTS} -cp "src/main/resources/:target/classes:target/lib/*:target/dependency/*" macrobase.MacroBase batch $conf_file
+java ${JAVA_OPTS} -cp "src/main/resources/:target/classes:target/lib/*:target/dependency/*" macrobase.MacroBase pipeline $conf_file
```

Comment:
config file could be the first argument to the script, like it is now with `bin/batch.sh`. Otherwise they are identical, no?